lib/logger/throttler.go

```diff
@@ -52,16 +57,26 @@ func newLogThrottler(throttle time.Duration) *LogThrottler {
 func (lt *LogThrottler) Errorf(format string, args ...any) {
 	select {
 	case lt.ch <- struct{}{}:
+		dropped := atomic.SwapUint64(&lt.dropped, 0)
+		if dropped > 0 {
```

Comment:
I think it's fine to print the message when dropped==0 as
```
Similar 0 log messages were dropped due to throttling
```
It shouldn't bring confusion, but keeps consistency and helps with log filtering. wdyt?